airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/DefaultJdbcDatabase.java

```diff
@@ -73,10 +74,13 @@ public <T> Stream<T> unsafeResultSetQuery(final CheckedFunction<Connection, Resu
 
   @Override
   public DatabaseMetaData getMetaData() throws SQLException {
-    final Connection conn = dataSource.getConnection();
-    final DatabaseMetaData metaData = conn.getMetaData();
-    conn.close();
-    return metaData;
+    try (final Connection connection = dataSource.getConnection()) {
+      final DatabaseMetaData metaData = connection.getMetaData();
+      connection.close();
```

Comment:
I think it's redundant to close the connection here since Connection extends AutoClosable